src/manifest.json

```
@@ -1,7 +1,7 @@
 {
     "manifest_version": 2,
     "name": "Open in VSCode",
-    "version": "1.1.0",
+    "version": "1.2.0",
```

```suggestion
    "version": "1.1.0",
```

I'll do this in the main branch if the PR passes through